src/mmc_windows.c

```diff
@@ -166,10 +166,12 @@ static void unmount_volume(LPWSTR volume_name) {
                              &bytes_returned,
                              NULL);
 
-    CloseHandle(volume_handle);
 
     if (!status)
-        fwup_errx(EXIT_FAILURE, "Error unmounting '%S' (Error %lu)", volume_name, GetLastError());
+        fwup_errx(EXIT_FAILURE, "Error unmounting '%S' (Error %lu)", volume_name, GetLastError());   
+
+    // Note we deliberately do not FSCTL_UNLOCK_VOLUME here, as the Logical Volume must
```

Comment:
Could you change "we deliberately do not FSCTL_UNLOCK_VOLUME" to "we deliberately do not FSCTL_UNLOCK_VOLUME or call CloseHandle"?

I agree, this looks really weird.